emails/views.py

```diff
@@ -322,6 +341,7 @@ def _get_text_and_html_content(email_message):
                 if part.get_content_type() == 'text/html':
                     html_content = content
                 if part.is_attachment():
+                    has_attachment = True
                     incr_if_enabled('email_with_attachment', 1)
```

Comment:
I would like it if we can pass how big the attachment is but not sure how to best do that. Ideally we are not just incrementing the metric but having a record for each attachment size. What do you think @groovecoder?

Reply:
Yeah, that'd be good. In theory the email could include a `Content-Length` header in addition to its `Content-Disposition` header?